### PR TITLE
Support nocopy option in volume binds (#510)

### PR DESCRIPTION
**Pull request by dmandalidis**

### CODE REVIEW

File: src/main/java/com/spotify/docker/client/messages/HostConfig.java [hunk 1]
```diff
@@ -1036,7 +1038,12 @@ public String toString() {
       } else if (from == null || from.equals("")) {
         return to;
       } else if (readOnly == null || !readOnly) {
-        return from + ":" + to;
+          if (noCopy == null || !noCopy) {
```
Comment (johnflavin) [inline src/main/java/com/spotify/docker/client/messages/HostConfig.java]: Are `noCopy` and `readOnly` mutually exclusive? As in, you can only ever have `from:to:ro` or `from:to:nocopy` but never `from:to:ro:nocopy`?

Comment (dmandalidis) [inline src/main/java/com/spotify/docker/client/messages/HostConfig.java]: Yeap (doesn't make sense after all)

`mandas@docker-1:~$ docker volume create --name foo
foo

mandas@docker-1:~$ docker run -v foo:/tmp:ro:nocopy busybox
docker: Error response from daemon: Invalid bind mount spec "foo:/tmp:ro:nocopy": Invalid volume specification: 'foo:/tmp:ro:nocopy'.
See 'docker run --help'.
`

Comment (dmandalidis) [inline src/main/java/com/spotify/docker/client/messages/HostConfig.java]: Phew! This option is cursed, it is supported in bind mounts although the man page says it so :

https://github.com/docker/docker/issues/27046

Comment (johnflavin) [inline src/main/java/com/spotify/docker/client/messages/HostConfig.java]: I checked out [the volume options](https://docs.docker.com/engine/reference/run/#/volume-shared-filesystems) after reading docker/docker#27046. It isn't worded very clearly, but it does say you can set `ro` and `nocopy` together, along with any of the other options.
> The **comma-delimited** `options` are are `[rw|ro]`, `[z|Z]`, `[[r]shared|[r]slave|[r]private]`, and `[nocopy]`.

Emphasis mine.


```bash
$ docker volume create --name foo
foo
$ docker run -v foo:/tmp:ro:nocopy busybox
docker: Error response from daemon: Invalid bind mount spec "foo:/tmp:ro:nocopy": Invalid volume specification: 'foo:/tmp:ro:nocopy'.
See 'docker run --help'.
$ docker run -v foo:/tmp:ro,nocopy busybox
[no error]
```

Comment (dmandalidis) [inline src/main/java/com/spotify/docker/client/messages/HostConfig.java]: So probably, 'ro' has to do with the operation and 'nocopy' with the initialization...I see

**Comment by dmandalidis** [comment]
Ok but I think this goes to another PR (I could take care of this)

**Pull request merged**